packages/nimble-components/src/table/styles.ts

```diff
@@ -150,6 +150,7 @@ export const styles = css`
         cursor: col-resize;
         position: absolute;
         z-index: ${ZIndexLevels.zIndex1};
+        touch-action: pan-y;
```

Comment:
This makes it so that the column dividers only support panning up/down by touch-dragging. We need this to prevent the browser from hijacking the horizontal drag (sends us the `pointercancel` event) to do a horizontal pan.